spec/rails_helper.rb

```diff
@@ -4,6 +4,7 @@
 require File.expand_path('../../config/environment', __FILE__)
 require 'rspec/rails'
 # Add additional requires below this line. Rails is not loaded until this point!
+require 'webmock/rspec'
```

Comment:
For now I'm using it to assert statuses are created on GitHub, but I foresee more uses of it on the way.